rsmtool/rsmexplain.py

```diff
@@ -345,10 +349,13 @@ def generate_explanation(
         range_size = int(configuration.get("sample_size"))
     elif has_sample_range:
         range_size = parse_range(configuration.get("sample_range"))
+    elif has_sample_ids:
+        range_size = configuration.get("sample_ids").split(",")
+        range_size = tuple([int(id) for id in range_size])
```

Comment:
```suggestion
        range_size = tuple([int(id_) for id_ in range_size])
```

`id` is a reserved keyword in Python that tells you the memory address of an object. It's probably not a big deal to override it in the scope of a list comprehensive but it's always better to be safe.